### PR TITLE
workaround: add instruction for err

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve? 
add instruction for some error

### What is changed and how it works?
add instruction in error.workaround

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: pkg/terror/error_list.go [hunk 1]
```diff
@@ -560,28 +560,28 @@ var (
...
-	ErrVerifySQLOperateArgs   = New(codeVerifySQLOperateArgs, ClassFunctional, ScopeInternal, LevelLow, "", "")
+	ErrVerifySQLOperateArgs   = New(codeVerifySQLOperateArgs, ClassFunctional, ScopeInternal, LevelLow, "", "Please make sure the args are correct.")
 	ErrStatFileSize           = New(codeStatFileSize, ClassFunctional, ScopeInternal, LevelHigh, "get file statfs", "")
 	ErrReaderAlreadyRunning   = New(codeReaderAlreadyRunning, ClassFunctional, ScopeInternal, LevelHigh, "binlog reader is already running", "")
 	ErrReaderAlreadyStarted   = New(codeReaderAlreadyStarted, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s, already started", "")
 	ErrReaderStateCannotClose = New(codeReaderStateCannotClose, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s, can not close", "")
-	ErrReaderShouldStartSync  = New(codeReaderShouldStartSync, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s, please start sync first", "")
+	ErrReaderShouldStartSync  = New(codeReaderShouldStartSync, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s", "Please start sync first.")
```
Comment (WangXiangUSTC) [inline pkg/terror/error_list.go:582]: seems that this is DM's internal error, users can't resolve it

File: pkg/terror/error_list.go [hunk 2]
```diff
@@ -646,7 +646,7 @@ var (
 	ErrBinlogReadFileByGTID          = New(codeBinlogReadFileByGTID, ClassFunctional, ScopeInternal, LevelHigh, "read from file by GTID not supported", "")
 	ErrBinlogWriterNotStateNew       = New(codeBinlogWriterNotStateNew, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s, already started", "")
 	ErrBinlogWriterStateCannotClose  = New(codeBinlogWriterStateCannotClose, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s, can not close", "")
-	ErrBinlogWriterNeedStart         = New(codeBinlogWriterNeedStart, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s, please start the writer first", "")
+	ErrBinlogWriterNeedStart         = New(codeBinlogWriterNeedStart, ClassFunctional, ScopeInternal, LevelHigh, "stage %s, expect %s", "Please start the writer first.")
```
Comment (WangXiangUSTC) [inline pkg/terror/error_list.go:649]: ditto

File: pkg/terror/error_list.go [hunk 3]
```diff
@@ -684,42 +684,42 @@ var (
...
-	ErrConfigTomlTransform          = New(codeConfigTomlTransform, ClassConfig, ScopeInternal, LevelMedium, "%s", "")
-	ErrConfigTaskYamlTransform      = New(codeConfigTaskYamlTransform, ClassConfig, ScopeInternal, LevelMedium, "%s", "")
-	ErrConfigTaskNameEmpty          = New(codeConfigTaskNameEmpty, ClassConfig, ScopeInternal, LevelMedium, "task name should not be empty", "")
-	ErrConfigEmptySourceID          = New(codeConfigEmptySourceID, ClassConfig, ScopeInternal, LevelMedium, "empty source-id not valid", "")
-	ErrConfigTooLongSourceID        = New(codeConfigTooLongSourceID, ClassConfig, ScopeInternal, LevelMedium, "too long source-id not valid", "")
-	ErrConfigOnlineSchemeNotSupport = New(codeConfigOnlineSchemeNotSupport, ClassConfig, ScopeInternal, LevelMedium, "online scheme %s not supported", "")
-	ErrConfigInvalidTimezone        = New(codeConfigInvalidTimezone, ClassConfig, ScopeInternal, LevelMedium, "invalid timezone string: %s", "")
+	ErrConfigCheckItemNotSupport    = New(codeConfigCheckItemNotSupport, ClassConfig, ScopeInternal, LevelMedium, "checking item %s is not supported\n%s", "Please check `ignore-checking-items` config in task configuration file.")
```
Comment (WangXiangUSTC) [inline pkg/terror/error_list.go:690]: how about also provide the items which are supported?

File: pkg/terror/error_list.go [hunk 4]
```diff
@@ -684,42 +684,42 @@ var (
...
+	ErrConfigTomlTransform          = New(codeConfigTomlTransform, ClassConfig, ScopeInternal, LevelMedium, "%s", "Please check the configuration file has correct TOML format.")
+	ErrConfigTaskYamlTransform      = New(codeConfigTaskYamlTransform, ClassConfig, ScopeInternal, LevelMedium, "%s", "Please check the configuration file has correct YAML format.")
+	ErrConfigTaskNameEmpty          = New(codeConfigTaskNameEmpty, ClassConfig, ScopeInternal, LevelMedium, "task name should not be empty", "Please check the `name` config in task configuration file.")
+	ErrConfigEmptySourceID          = New(codeConfigEmptySourceID, ClassConfig, ScopeInternal, LevelMedium, "empty source-id not valid", "Please check the `source-id` config in configuration file.")
+	ErrConfigTooLongSourceID        = New(codeConfigTooLongSourceID, ClassConfig, ScopeInternal, LevelMedium, "too long source-id not valid", "Please check the `source-id` config in configuration file.")
```
Comment (WangXiangUSTC) [inline pkg/terror/error_list.go:695]: how about providing the max source id length?

File: pkg/terror/error_list.go [hunk 5]
```diff
@@ -684,42 +684,42 @@ var (
...
+	ErrConfigOnlineSchemeNotSupport = New(codeConfigOnlineSchemeNotSupport, ClassConfig, ScopeInternal, LevelMedium, "online scheme %s not supported", "Please check the `online-ddl-scheme` config in task configuration file.")
```
Comment (WangXiangUSTC) [inline pkg/terror/error_list.go:696]: how about also provide the supported config, `gh-ost` and `pt`

File: pkg/terror/error_list.go [hunk 6]
```diff
@@ -684,42 +684,42 @@ var (
...
+	ErrConfigSyncerCfgNotFound      = New(codeConfigSyncerCfgNotFound, ClassConfig, ScopeInternal, LevelMedium, "mysql-instance(%d)'s syncer config %s not exist in syncer", "Please check the `syncer-config-name` config in task configuration file.")
+	ErrConfigSourceIDNotFound       = New(codeConfigSourceIDNotFound, ClassConfig, ScopeInternal, LevelMedium, "source %s in deployment configuration not found", "Please use `operate-source create source-config-file-path` to add source.")
+	ErrConfigDuplicateCfgItem       = New(codeConfigDuplicateCfgItem, ClassConfig, ScopeInternal, LevelMedium, "the following mysql configs have duplicate items, please remove the duplicates:\n%s", "Please check the `mysql-instances` config in task configuration file.")
+	ErrConfigShardModeNotSupport    = New(codeConfigShardModeNotSupport, ClassConfig, ScopeInternal, LevelMedium, "shard mode %s not supported", "Please check the `shard-mode` config in task configuration file.")
```
Comment (WangXiangUSTC) [inline pkg/terror/error_list.go:722]: how about providing the mode we supported, `pessimistic` and `optimistic`

File: _utils/terror_gen/errors_release.txt [hunk 7]
```diff
@@ -109,45 +109,45 @@ ErrTracingGetTSO,[code=11102:class=functional:scope=internal:level=high], "msg:
...
+ErrConfigMySQLInstSameSourceID,[code=20014:class=config:scope=internal:level=medium], "msg: 'mysql-instance (%d) and (%d) have same source-id (%s)', workaround: 'Please check the `mysql-instances` config in task configuration file.'"
+ErrConfigMydumperCfgConflict,[code=20015:class=config:scope=internal:level=medium], "msg: 'mydumper-config-name and mydumper should only specify one', workaround: 'Please check the `mydumper-config-name` and `mydumper` config in task configuration file.'"
+ErrConfigLoaderCfgConflict,[code=20016:class=config:scope=internal:level=medium], "msg: 'loader-config-name and loader should only specify one', workaround: 'Please check the `loader-config-name` and `loader` config in task configuration file.'"
+ErrConfigSyncerCfgConflict,[code=20017:class=config:scope=internal:level=medium], "msg: 'syncer-config-name and syncer should only specify one', workaround: 'Please check the `syncer-config-name` and `loader` config in task configuration file.'"
```
Comment (lance6716) [inline _utils/terror_gen/errors_release.txt:134]: "loader" is "syncer"?

File: errors.toml [hunk 8]
```diff
@@ -763,43 +763,43 @@ tags = ["internal", "medium"]
...
 description = ""
-workaround = ""
+workaround = "Please check the `loader-config-name` and `loader` config in task configuration file."
 tags = ["internal", "medium"]
 
 [error.DM-config-20017]
 message = "syncer-config-name and syncer should only specify one"
 description = ""
-workaround = ""
+workaround = "Please check the `syncer-config-name` and `loader` config in task configuration file."
```
Comment (lance6716) [inline errors.toml:802]: ditto

File: _utils/terror_gen/errors_release.txt (hunk 7 above)
Comment (GMHDBJD) [reply _utils/terror_gen/errors_release.txt]: address in f9a35b9

**Pull request merged**